lib/Firtool/Firtool.cpp

```diff
@@ -78,7 +78,7 @@ LogicalResult firtool::populateCHIRRTLToLowFIRRTL(mlir::PassManager &pm,
   pm.nest<firrtl::CircuitOp>().addPass(firrtl::createInferWidthsPass());
 
   pm.nest<firrtl::CircuitOp>().addPass(
-      firrtl::createMemToRegOfVecPass(opt.shouldReplicateSequentialMemories(),
+      firrtl::createMemToRegOfVecPass(opt.shouldReplaceSequentialMemories(), 
```

Comment:
the formatter doesn't like the trailing whitespace at the end of this line

Reply:
Hi Megan! Yea i just saw this, i made the change just now. Had to install the clang formatter. Should be good now? We'll see what CI job says

Reply:
I'll have to doublecheck the config file, I didn't see your comments till now (before I pushed the newest change to get rid of the whitespace). I'll check my settings and formatting now. Thanks :)